packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -1479,7 +1495,7 @@ describe('TransactionController', () => {
     });
   });
 
-  describe('queryTransactionStatus', () => {
+  describe.skip('queryTransactionStatus', () => {
```

Comment:
These tests are broken in the commit https://github.com/MetaMask/core/commit/6d4b56234713195e81da7cd0c01d352dfdb1a999
I couldn't find a easy fix at first look 
cc: @tommasini